app/views/works/_edit_javascript.html.erb

```diff
@@ -81,30 +81,122 @@
       return html;
     }
 
+    // ************************************************ //
     // Related Objects
     // related_identifier:, related_identifier_type:, relation_type
     var addRelatedObjectHtml = function(num, related_identifier, related_identifier_type, relation_type) {
       var rowId = `related_object_row_${num}`;
       var relatedIdentifierId = `related_identifier_${num}`;
       var relatedIdentifierTypeId = `related_identifier_type_${num}`;
       var relationTypeId = `relation_type_${num}`;
+      var relatedIdentifierTypeHtml = relatedIdentififerTypeSelectHtml(relatedIdentifierTypeId, related_identifier_type);
+      var relationTypeHtml = relationTypeSelectHtml(relationTypeId, relation_type);
 
       var rowHtml = `<tr id="${rowId}" class="related-objects-table-row">
         <td>
           <input type="text" id="${relatedIdentifierId}" name="${relatedIdentifierId}" value="${related_identifier}" data-num="${num}" placeholder="The URL web address for a related publication or other resource" />
         </td>
         <td>
-          <input type="text" id="${relatedIdentifierTypeId}" name="${relatedIdentifierTypeId}" value="${related_identifier_type}" />
+          ${relatedIdentifierTypeHtml}
         </td>
         <td>
-          <input type="text" id="${relationTypeId}" name="${relationTypeId}" value="${relation_type}" />
+          ${relationTypeHtml}
         </td>
       </tr>`;
       $("#related-objects-table").append(rowHtml);
     }
 
+    var relatedIdentififerTypeSelectHtml = function(relatedIdentifierTypeId, related_identifier_type) {
+      var i, html, related_identifier_type_options, key, value;
```

Comment:
Unless we're expecting to mutate these, using `const` can make sure we don't do it by mistake.

Reply:
is `related_identifier_type_options` used?